components/Navigation.js

```diff
@@ -0,0 +1,57 @@
+import Link from "next/link";
+import { useRouter } from "next/router";
+import styled from "styled-components";
+
+const Navigation = () => {
+  const router = useRouter();
+
+  return (
+    <NavBar>
+      <StyledLink href="/" $active={router.pathname === "/"}>
+        I
+      </StyledLink>
+      <StyledLink href="/pairings" $active={router.pathname === "/pairings"}>
+        P
+      </StyledLink>
```

Comment:
This Link leads to a non-existing page. Therefore, you should remove the Link.

Once you build the pairings page it makes sense to re-introduce this link.

```suggestion
```